pkg/health/checker/checker.go

```diff
@@ -176,10 +221,42 @@ func publishLatestHealth(inCh <-chan api.KVPairs, quitCh <-chan struct{}, result
 	return errCh
 }
 
+func (h healthChecker) getStatusEndpoints(
+	serviceID string,
+	status manifest.StatusStanza,
+) ([]string, error) {
+	rcIDs, err := h.resourceClient.GetRCIDsForPod(types.PodID(serviceID))
+	if err != nil {
+		return nil, err
+	}
+	var statusEndpoints []string
+	for _, rcID := range rcIDs {
+		rc, err := h.rcStore.Get(rcID)
+		if err != nil {
+			return nil, err
+		}
+		// get hostnames for rc
+		labeled, err := h.labelReader.GetMatches(rc.NodeSelector, labels.NODE)
+		if err != nil {
+			return nil, err
+		}
+		// create status endpoints
+		statusEndpoints = make([]string, 0, len(labeled))
+		statusPath := status.Path
+		if status.Path == "" {
+			statusPath = "_status"
```

Comment:
nit: this might be a nice detail to hide in the status stanza struct, e.g. `statusStanza.Path()` and it applies the default if necessary